test/Stateless.Tests/AsyncFireingModesFixture.cs

```diff
@@ -1,80 +1,80 @@
-﻿#if TASKS
-using System.Collections.Generic;
-using Xunit;
+﻿//#if TASKS
+//using System.Collections.Generic;
+//using Xunit;
 
 
-namespace Stateless.Tests
-{
-    /// <summary>
-    /// This test class verifies that the firing modes are working as expected
-    /// </summary>
-    public class AsyncFireingModesFixture
-    {
-        /// <summary>
-        /// Check that the immediate fireing modes executes entry/exit out of order.
-        /// </summary>
-        [Fact]
-        public void ImmediateEntryAProcessedBeforeEnterB()
-        {
-            var record = new List<string>();
-            var sm = new StateMachine<State, Trigger>(State.A, FiringMode.Immediate);
+//namespace Stateless.Tests
```

Comment:
Any reason that these tests are commented out? Cheers!

Reply:
done: uncommented again, all tests green